recipes/bttoxin_digger/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - perl-pod-usage
     - perl-getopt-long
     - perl-file-tee
-    - pgcgap >=1.0.14
+    - pgcgap >=1.0.14,<1.0.23
```

Comment:
do you need to restrict this to <1.0.23 ?

Reply:
Yes, it is necessary for BtToxin_Digger v1.0.5, in the next version, the PGCGAP must be version 1.0.23 or higher. Thank you very much!